e2e/test/scenarios/sharing/public-resource-downloads.cy.spec.ts

```diff
@@ -68,6 +76,12 @@ describeEE(
         cy.get("header").findByText("Export as PDF").click();
 
         cy.verifyDownload("Orders in a dashboard.pdf");
+
+        expectGoodSnowplowEvent({
+          event: "dashboard_pdf_exported",
+          dashboard_id: 0,
```

Comment:
seems we also don't have `dashboard_id` not only for static dashboard, but also public dashboards.

Technically we know this value on BE, but I guess we don't care much in this case?

Reply:
Yep we don't want them, the ids are either jwt or uuid and we don't want to pass those, let me clarify that better in the code comments

Reply:
Is it better with this new comment?
https://github.com/metabase/metabase/pull/46335/commits/940b45cb4457f4836842ec9ae65412f0372d8793

Reply:
Looks good to me.